libraries/AP_HAL_ChibiOS/hwdef/ZubaxGNSS/hwdef.dat

```diff
@@ -20,7 +20,7 @@ define STORAGE_FLASH_PAGE 126
 define HAL_STORAGE_SIZE 800
 
 # board ID for firmware load
-APJ_BOARD_ID 3
+APJ_BOARD_ID 1003
```

Comment:
Some explanation here of the composition of the board ID might be a good idea for those looking for things to copy later.